lib/timelines/models/concerns/has_events.rb

```diff
@@ -0,0 +1,11 @@
+require "active_support/concern"
+
+module Timelines
+  module HasEvents
+    extend ActiveSupport::Concern
+
+    included do
+      has_many :events
```

Comment:
Not used concerns before. What are we looking to achieve here, how do I test that I've set this up properly in the Rails console?

Reply:
This looks good, but you'll need to specify a class name for the has_many: 
`has_many :events, class_name: "Timelines::Event", as: :resource, dependent: :destroy`